presto-hive/src/main/java/com/facebook/presto/hive/HiveZeroRowFileCreator.java

```diff
@@ -107,10 +108,12 @@ private byte[] generateZeroRowFile(
     {
         String tmpDirectoryPath = System.getProperty("java.io.tmpdir");
         String tmpFileName = format("presto-hive-zero-row-file-creator-%s-%s", session.getQueryId(), randomUUID().toString());
-        java.nio.file.Path tmpFilePath = Paths.get(tmpDirectoryPath, tmpFileName);
+        String normalizedDirPath = Normalizer.normalize(tmpDirectoryPath, Normalizer.Form.NFKC);
+        String normalizedFileName = Normalizer.normalize(tmpFileName, Normalizer.Form.NFKC);
```

Comment:
`tmpFileName` shouldn't need normalization since the the query-id and randomUUID should be regular ASCII characters as per design

Reply:
@aaneja As discussed marking the issue as false postive and closing this PR/